src/db.rs

```diff
@@ -75,8 +76,9 @@ pub async fn network_capacity(db: &Client) -> Result<Capacity> {
         .map_err(Debug)
 }
 
-// Return all the hosts stored in `holoports_status` collection
-pub async fn list_all_hosts(db: &Client) -> Result<Vec<HostSummary>> {
+// Return the most recent record for hosts stored in `holoports_status` collection that have a successful SSH record
+// Ignores records older than 7 days
+pub async fn list_available_hosts(db: &Client, cutoff: u64) -> Result<Vec<HostSummary>> {
```

Comment:
`cutoff` seems to be a variable, so I would say:
```suggestion
// Ignores records older than <cutoff> days
pub async fn list_available_hosts(db: &Client, cutoff: u64) -> Result<Vec<HostSummary>> {
```